src/Microsoft.VisualStudio.Composition.AppHost/CreateComposition.cs

```diff
@@ -380,13 +401,15 @@ private static bool Probe(string filePath, Version minimumVersion, out Assembly
                     try
                     {
                         assembly = Assembly.Load(name);
+                        this.LogAssemblyLoad(assembly, filePath);
                         return true;
                     }
                     catch (BadImageFormatException)
                     {
                         // This happens for some reference assemblies that can only be loaded in a reflection-only context.
                         // But we're not allowed in an assembly resolve event to return reflection-only loaded assemblies.
                         // So just return false to communicate the failure.
```

Comment:
You might modify the comment here slightly to say "So just log it and return false ..."

Reply:
Done